modules/ROOT/pages/clauses/index.adoc

```diff
@@ -258,36 +229,69 @@ m| xref:clauses/transaction-clauses.adoc#query-terminate-transactions[TERMINATE
 
 |===
 
-[[writing-clauses]]
-== Writing clauses
 
-These comprise clauses that write the data to the database.
+[[reading-hints]]
+== Reading hints
+
+These comprise clauses used to specify planner hints when tuning a query.
+More details regarding the usage of these -- and query tuning in general -- can be found in xref::query-tuning/using.adoc[Planner hints and the USING keyword].
 
 [options="header"]
 |===
-| Clause | Description
+| Hint | Description
 
-m| xref::clauses/create.adoc[CREATE]
-| Create nodes and relationships.
+m| xref::query-tuning/using.adoc#query-using-index-hint[USING INDEX]
+| Index hints are used to specify which index, if any, the planner should use as a starting point.
 
-m| xref::clauses/delete.adoc[DELETE]
-a|
-Delete nodes, relationships or paths.
-Any node to be deleted must also have all associated relationships explicitly deleted.
+m| xref::query-tuning/using.adoc#query-using-index-hint[USING INDEX SEEK]
+| Index seek hint instructs the planner to use an index seek for this clause.
 
-m| xref::clauses/delete.adoc[DETACH DELETE]
-a|
-Delete a node or set of nodes.
-All associated relationships will automatically be deleted.
+m| xref::query-tuning/using.adoc#query-using-scan-hint[USING SCAN]
+| Scan hints are used to force the planner to do a label scan (followed by a filtering operation) instead of using an index.
 
-m| xref::clauses/set.adoc[SET]
-| Update labels on nodes and properties on nodes and relationships.
+m| xref::query-tuning/using.adoc#query-using-join-hint[USING JOIN]
+| Join hints are used to enforce a join operation at specified points.
 
-m| xref::clauses/remove.adoc[REMOVE]
-| Remove properties and labels from nodes and relationships.
+|===
 
-m| xref::clauses/foreach.adoc[FOREACH]
-| Update data within a list, whether components of a path, or the result of aggregation.
+[[index-and-constraint-clauses]]
+== Index and constraint clauses
+
+These comprise clauses to create, show, and drop indexes and constraints.
+
+[options="header"]
+|===
+| Clause | Description
 
+m| xref::indexes-for-search-performance.adoc#indexes-syntax[CREATE \| SHOW  \| DROP INDEX]
+| Create, show or drop an index.
+
+m| xref::constraints/syntax.adoc[CREATE \| SHOW \| DROP CONSTRAINT]
+| Create, show or drop a constraint.
+|===
+
+[[administration-clauses]]
+== Administration clauses
+
+These comprise clauses used to manage databases, schema and security; further details can found in xref::administration/databases.adoc[Database management] and xref::administration/access-control/index.adoc[Access control].
+
+[options="header"]
 |===
+| Clause | Description
+
+m| xref::administration/databases.adoc[CREATE \| SHOW \| START \| ALTER \| DROP \|STOP [COMPOSITE\] DATABASE]
```

Comment:
```suggestion
m| xref::administration/databases.adoc[CREATE \| SHOW \| START \| ALTER \| DROP \| STOP [COMPOSITE\] DATABASE]
```
Also, should STOP come before DROP? (drop seems to be always last otherwise)
```suggestion
m| xref::administration/databases.adoc[CREATE \| SHOW \| START \| ALTER \| STOP \| DROP [COMPOSITE\] DATABASE]
```